Clarified visibility rules for default Orgs and Teams (#6884)
TykTechnologies/tyk-docs: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/manage-api-consumer-organisations.md

- Move users from it to other Organisations as needed
- Use it as a holding area for users awaiting proper Organisation assignment

#### Developer App visibility

> [Team and Organisation level visibility]({{< ref "portal/developer-app#visibility" >}}) are not applied within the Default Organisation. This behavior has been implemented to ensure no accidental exposure of Developer Apps if a user is removed from a custom Organisation and reverts automatically to the Default Org.

Reviewer: ```suggestion
[Team and Organisation level app visibility]({{< ref "portal/developer-app#visibility" >}}) is not applied within the Default Organisation. This behavior has been implemented to prevent accidental exposure of Developer Apps if a user is removed from a custom Organisation and automatically reverts to the Default Org.
```

- Provides a home for API Consumer users who have been [removed]({{< ref "tyk-stack/tyk-developer-portal/enterprise-developer-portal/managing-access/manage-api-consumer-organisations#deleting-organisations" >}}) from all other Teams in the Org
- Can be used for Organisation-wide API access

The Default Team cannot be deleted, however you can:

- Rename it to better reflect your business needs
- Move users from it to other Teams as needed
- Use it as a holding area for users awaiting proper Team assignment

#### Developer App visibility

> [Team level visibility]({{< ref "portal/developer-app#visibility" >}}) is not applied within the Default Team. This behavior has been implemented to ensure no accidental exposure of Developer Apps if a user is removed from a team and reverts automatically to the Default Team.

Reviewer: ```suggestion
[Team level app visibility]({{< ref "portal/developer-app#visibility" >}}) is not applied within the Default Team. This behavior has been implemented to prevent accidental exposure of Developer Apps if a user is removed from a team and automatically reverts to the Default Team.
```
